src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -58,26 +82,24 @@ This configuration option defines how your Astro project is deployed to Cloudfla
 
 Switching to directory mode allows you to add additional files manually such as [Cloudflare Pages Plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/), [Cloudflare Pages Middleware](https://developers.cloudflare.com/pages/platform/functions/middleware/) or custom functions using [Cloudflare Pages Functions Routing](https://developers.cloudflare.com/pages/platform/functions/routing/).
 
-```js
-// astro.config.mjs
+```js title="astro.config.mjs"
 export default defineConfig({
   adapter: cloudflare({ mode: 'directory' }),
 });
 ```
```

Comment:
I think it would make sense to highlight the word: `directory`
WDYT?

Reply:
If there's no `mode` enabled by default, then I'd suggest highlighting (or even adding) `mode: 'dictionary'` in its entirety. Happy with whatever you decide, and happy for either of you to make the code suggestion!

Reply:
IIRC the default setting is `mode: 'advanced'`, but would have to double check